docs/source/user/installing.rst

```diff
@@ -152,6 +152,9 @@ Then you can build and install Numba from the top level of the source tree::
 
     $ python setup.py install
 
+If you wish to run the test suite, see the instructions in
+:doc:`developer documentation <running-tests>`.
```

Comment:
This doesn't seem to render correctly in the RTD build, might need to be a `:ref:` ?